GDevelop.js/Bindings/Wrapper.cpp

```diff
@@ -377,6 +377,12 @@ void removeFromVectorParameterMetadata(std::vector<gd::ParameterMetadata> &vec,
   vec.erase(vec.begin() + pos);
 }
 
+void insertIntoVectorParameterMetadata(std::vector<gd::ParameterMetadata> &vec,
+                                       size_t pos,
+                                       const ParameterMetadata& parameterMetadata) {
+  vec.insert(vec.begin() + pos, parameterMetadata);
+}
```

Comment:
I know we should rather decare an object for it, but I didn't want to refactor everything now.

Reply:
That's fine! We can always refactor out this function later.